int16 keyframe_id
int16 vin 
int16 requester_vin
float32[] pose
#sonar_oculus/OculusPing img
sensor_msgs/PointCloud2 cloud #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